starWarsQuotesRepository.py

```diff
@@ -39,6 +40,15 @@ def __getQuotes(self, trilogy: str = None) -> List[str]:
 
         return result
 
+    def __processQuote(self, quote: str, input: str = None) -> str:
```

Comment:
Can you add a quick return here? Like this:

```
if not utils.isValidStr(input):
    return quote
```

Reply:
Done.